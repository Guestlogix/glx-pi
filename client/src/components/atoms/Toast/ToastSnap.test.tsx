import * as React from "react"
import * as Enzyme from "enzyme"
import * as Adapter from "enzyme-adapter-react-16"
import { Toast } from "."

Enzyme.configure({ adapter: new Adapter() })

describe("Snapshot for Toast", () => {
  it("Snapshot", () => {
    const wrapper = Enzyme.shallow(<Toast message={"ERROR"} />)
    expect(wrapper).toMatchSnapshot()
  })
})
