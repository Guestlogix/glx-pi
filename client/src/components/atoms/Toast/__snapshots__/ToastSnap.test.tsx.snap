// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot for Toast Snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Toast
    message="ERROR"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <ForwardRef
        onClick={[Function]}
        show={true}
      >
        ERROR
      </ForwardRef>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": "ERROR",
        "onClick": [Function],
        "show": true,
      },
      "ref": null,
      "rendered": "ERROR",
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": false,
          "rules": Array [
            "
  display: ",
            [Function],
            ";
  position: absolute;
  cursor: pointer;
  padding: 1rem;
  top: 0;
  left: 0;
  z-index: 4;
  width: 100%;
  background-color: ",
            [Function],
            ";
  color: #fff;
  box-sizing: border-box;
  box-shadow: none;

  animation: ",
            Keyframes {
              "id": "sc-keyframes-gVCwOh",
              "inject": [Function],
              "name": "gVCwOh",
              "rules": Array [
                "@-webkit-keyframes gVCwOh{from{-webkit-transform :translateY( -10rem);-ms-transform :translateY( -10rem);transform :translateY( -10rem);}to{-webkit-transform:translateY(0);-ms-transform:translateY(0);transform:translateY(0);}}",
                "@keyframes gVCwOh{from{-webkit-transform :translateY( -10rem);-ms-transform :translateY( -10rem);transform :translateY( -10rem);}to{-webkit-transform:translateY(0);-ms-transform:translateY(0);transform:translateY(0);}}",
              ],
              "toString": [Function],
            },
            " 0.4s ease-in, ",
            Keyframes {
              "id": "sc-keyframes-jWhTOz",
              "inject": [Function],
              "name": "jWhTOz",
              "rules": Array [
                "@-webkit-keyframes jWhTOz{from{-webkit-transform :translateY( 0rem);-ms-transform :translateY( 0rem);transform :translateY( 0rem);display:block;}to{-webkit-transform:translateY(-10rem);-ms-transform:translateY(-10rem);transform:translateY(-10rem);display:none;}}",
                "@keyframes jWhTOz{from{-webkit-transform :translateY( 0rem);-ms-transform :translateY( 0rem);transform :translateY( 0rem);display:block;}to{-webkit-transform:translateY(-10rem);-ms-transform:translateY(-10rem);transform:translateY(-10rem);display:none;}}",
              ],
              "toString": [Function],
            },
            " 0.4s ease-out 6s forwards;
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <ForwardRef
          onClick={[Function]}
          show={true}
        >
          ERROR
        </ForwardRef>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "ERROR",
          "onClick": [Function],
          "show": true,
        },
        "ref": null,
        "rendered": "ERROR",
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": false,
            "rules": Array [
              "
  display: ",
              [Function],
              ";
  position: absolute;
  cursor: pointer;
  padding: 1rem;
  top: 0;
  left: 0;
  z-index: 4;
  width: 100%;
  background-color: ",
              [Function],
              ";
  color: #fff;
  box-sizing: border-box;
  box-shadow: none;

  animation: ",
              Keyframes {
                "id": "sc-keyframes-gVCwOh",
                "inject": [Function],
                "name": "gVCwOh",
                "rules": Array [
                  "@-webkit-keyframes gVCwOh{from{-webkit-transform :translateY( -10rem);-ms-transform :translateY( -10rem);transform :translateY( -10rem);}to{-webkit-transform:translateY(0);-ms-transform:translateY(0);transform:translateY(0);}}",
                  "@keyframes gVCwOh{from{-webkit-transform :translateY( -10rem);-ms-transform :translateY( -10rem);transform :translateY( -10rem);}to{-webkit-transform:translateY(0);-ms-transform:translateY(0);transform:translateY(0);}}",
                ],
                "toString": [Function],
              },
              " 0.4s ease-in, ",
              Keyframes {
                "id": "sc-keyframes-jWhTOz",
                "inject": [Function],
                "name": "jWhTOz",
                "rules": Array [
                  "@-webkit-keyframes jWhTOz{from{-webkit-transform :translateY( 0rem);-ms-transform :translateY( 0rem);transform :translateY( 0rem);display:block;}to{-webkit-transform:translateY(-10rem);-ms-transform:translateY(-10rem);transform:translateY(-10rem);display:none;}}",
                  "@keyframes jWhTOz{from{-webkit-transform :translateY( 0rem);-ms-transform :translateY( 0rem);transform :translateY( 0rem);display:block;}to{-webkit-transform:translateY(-10rem);-ms-transform:translateY(-10rem);transform:translateY(-10rem);display:none;}}",
                ],
                "toString": [Function],
              },
              " 0.4s ease-out 6s forwards;
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
